<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreeEMS_LoaderClass</class>
 <widget class="QWidget" name="FreeEMS_LoaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeEMS_Loader</string>
  </property>
  <widget class="QPushButton" name="pushConnect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLoad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Bitstream Charter&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRip">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
   </property>
   <property name="text">
    <string>Rip</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioRX">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioTX">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TX</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>55</number>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDevice">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>221</width>
       <height>27</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkRip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Rip Before Load</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkVerify">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Verify Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource-root.qrc">:/images/images/logos.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Stop bits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBaud">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>141</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Baud rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>Data bits:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDataBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboStopBits">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Bitstream Charter'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Pairty:&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboParity">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>78</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QPushButton" name="pushAbout">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt &quot;Bitstream Charter&quot;;</string>
     </property>
     <property name="text">
      <string>More</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushErase">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Bitstream Charter&quot;;</string>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource-root.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushConnect</sender>
   <signal>released()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRip</sender>
   <signal>clicked()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>rip()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushLoad</sender>
   <signal>released()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushErase</sender>
   <signal>clicked()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>eraseFlash()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAbout</sender>
   <signal>clicked()</signal>
   <receiver>FreeEMS_LoaderClass</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal2()</signal>
  <slot>test()</slot>
  <slot>slot1()</slot>
  <slot>connect()</slot>
  <slot>rip()</slot>
  <slot>getFileName()</slot>
  <slot>slot2()</slot>
  <slot>slot4()</slot>
  <slot>eraseFlash()</slot>
  <slot>load()</slot>
  <slot>showAbout()</slot>
  <slot>slot5()</slot>
 </slots>
</ui>
